client/client-multi/client-c/src/kaa/kaa.c

```diff
@@ -30,6 +30,10 @@
 #include "kaa_defaults.h"
 #include "platform/ext_key_utils.h"
```

Comment:
The `ext_key_utils.h` is responsible for RSA keys and the new `ext_encryption_utils.h` file manages AES (session) key and performs encryption and decryption of data.